recipes/linopy/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "linopy" %}
+{% set version = "0.0.14" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/linopy-{{ version }}.tar.gz
+  sha256: 78cc948d4a4840f6ce12337a75eaee79b6064d4517321ae0b1b495ca85455227
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.8, <=3.10
+    - pip
+    - setuptools_scm
+  run:
+    - python >=3.7
```

Comment:
Is there any reason these two Python lines aren't the same?

Reply:
no, I forgot this one, thanks